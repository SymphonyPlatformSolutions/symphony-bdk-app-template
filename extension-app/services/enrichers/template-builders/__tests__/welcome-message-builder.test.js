import WelcomeMessageBuilder from '../welcome-message-builder';

describe('Welcome Message Builder', () => {
  it('Should build message', () => {
    expect(WelcomeMessageBuilder.build()).toMatchSnapshot();
  });
});
