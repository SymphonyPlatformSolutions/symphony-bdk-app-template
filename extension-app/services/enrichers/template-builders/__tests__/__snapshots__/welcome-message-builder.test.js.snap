// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome Message Builder Should build message 1`] = `
"<messageML>
  <card class=\\"barStyle\\" data-icon-src=\\"\\">
    <header>
      I'm the Template Bot. I'll help you to keep updated about what happens on your services on Template. Here's how to start interacting with me following the commands below:
      <br />
      <br />
      <div>
        <div style=\\"padding-bottom: 4px;\\"><b style=\\"color: \\">Commands</b></div>
        <div style=\\"border-left: 2px solid ;\\">
          <div style=\\"padding-left: 4px;\\">
            <strong>Help:</strong> used to help you with the commands available to interact with the Template Bot
            <br />
            <strong>Syntax:</strong> @Template bot <strong>help</strong>
          </div>
        </div>
      </div>
    </header>
  </card>
</messageML>"
`;
